registry-scanner/hack/create-release-pr.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+### This script creates a new release PR
+# - install gh cli and semver-cli (go install github.com/davidrjonas/semver-cli@latest)
+# - create and push "release-X.Y" branch
+# - checkout this branch locally
+# - run this script from repo root: ./hack/create-release-pr.sh [REMOTE]
+# - merge the PR
+# It will trigger the release workflow that would create release draft on github
+
+TARGET_VERSION="$1"
+set -eu
+set -o pipefail
+
+if test "${TARGET_VERSION}" = ""; then
+	echo "USAGE: $0 <version>" >&2
+	exit 1
+fi
+
+CURRENT_BRANCH="$(git branch --show-current)"
+SUBMODULE_NAME="registry-scanner"
+
+if [[ ! "$CURRENT_BRANCH" == release-* ]]; then
+	echo "!! Please checkout branch 'release-X.Y' (currently in branch: '${CURRENT_BRANCH}')" >&2
```

Comment:
Looks like the required release branch name follows pattern like `registry-scanner/release-0.15`. If so, the above 2 lines need change to reflect that.